app/services/scenarios_fetch_service.rb

```diff
@@ -53,7 +54,7 @@ def self.fetch_all
       Rails.logger.debug(filename)
 
       # https://github.com/ruby/psych/issues/262
-      scenarios_list = YAML.load(File.read(filename))
+      scenarios_list = YAML.safe_load(File.read(filename))
```

Comment:
the `safe_load` throws errors when a yaml key looksl ike a date - are you sure this works? the link on line 56 has more info.

Reply:
@Br3nda hrm true good point, I will change it back just in case. This was a Code Climate recommendation but I didn't know that about the dates!